src/Runner.Common/RunServer.cs

```diff
@@ -20,40 +21,17 @@ public interface IRunServer : IRunnerService
     public sealed class RunServer : RunnerService, IRunServer
```

Comment:
Looks misleading, but few things happened here:
- `RunServer` is actually now [`ActionsRunServer`](https://github.com/actions/runner/pull/2158/files#diff-c8ce56f6157f18d7c9c8327441cf713e01ba63f4298973db5feab47d3089ad5e)
   - I could have made git consider this as rename I guess, but not a large file, so hopefully this reads okay
 -   `EstablishVssConnection` and `RetryRequest` to base class [`RunnerService`](https://github.com/actions/runner/pull/2158/files#diff-2083ec27a6019ab7168a40429dada422033d3b97b1f821c6022a0a7be34294e0) so that these can be shared with the this new `RunServer` (this file), which is just responsible for calling run service directly (no proxying through actions service)